pieces/TextSummarizerPiece/piece.py

```diff
@@ -1,11 +1,19 @@
 from domino.base_piece import BasePiece
 from .models import InputModel, OutputModel
 from typing import List
+from enum import Enum
 import openai
 import tiktoken
 import asyncio
 
 
+class TokenLimit(int, Enum):
```

Comment:
@luiztauffer, I've created this class to map the token limit for each OpenAI model. If you have any better ideas, please let me know.

Reply:
looks good to me